.nxignore

```diff
@@ -17,6 +17,7 @@ packages/fluentui/.browserslistrc
 **/fixtures/**
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33059&commitId=ee7bbb7c62594d46b5414b9bcf3ce279de2302e5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|3279| Changed|
|Drawer.overlay drawer full.chromium.png|1138| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>